flatland_viz/src/flatland_viz.cpp

```diff
@@ -80,9 +109,32 @@ FlatlandViz::FlatlandViz(FlatlandWindow* parent) : QWidget((QWidget*)parent) {
   // librviz.
   manager_ = new rviz::VisualizationManager(render_panel_);
   render_panel_->initialize(manager_->getSceneManager(), manager_);
+
+  // bind toolbar events
+  rviz::ToolManager* tool_man = manager_->getToolManager();
+
+  connect(manager_, SIGNAL(configChanged()), this,
+          SLOT(setDisplayConfigModified()));
+  connect(tool_man, &rviz::ToolManager::toolAdded, this, &FlatlandViz::addTool);
+  connect(tool_man, SIGNAL(toolRemoved(rviz::Tool*)), this,
+          SLOT(removeTool(rviz::Tool*)));
+  connect(tool_man, SIGNAL(toolRefreshed(rviz::Tool*)), this,
+          SLOT(refreshTool(rviz::Tool*)));
+  connect(tool_man, SIGNAL(toolChanged(rviz::Tool*)), this,
+          SLOT(indicateToolIsCurrent(rviz::Tool*)));
+
   manager_->initialize();
+
+  tool_man->addTool("flatland_viz/SpawnModel");
+
   manager_->startUpdate();
 
+  // Q_EMIT toolAdded( tool );
+  // addTool( "rviz/Interact" );
+  // addTool( "rviz/Select" );
+  // addTool( "rviz/SetInitialPose" );
+  // addTool( "rviz/SetGoal" );
```

Comment:
more test/debug commented out code to clean up

Reply:
I think we should start with default tools for Rviz plus Spawn Model Tool.